flent/tests/tcp_cc_surveil.inc

```diff
@@ -0,0 +1,46 @@
+## -*- mode: python; coding: utf-8 -*-
+
+# Mixin include file to add surveilling pid flows to a test
+
+PIDS=get_test_parameter('socket_stats_pids')
+HOST="localhost"
+DEFAULTS={'PLOT': 'tcp_cwnd'}
+
+if PIDS:
+    for pid in PIDS.split(","):
+        DATA_SETS['sock_stats::%s' % pid] = {'command': find_ss_iterate(IP_VERSION, HOST, STEP_SIZE, TOTAL_LENGTH),
+                                              'units': 'misc',
+                                              'runner': 'ss',}
+
+PLOTS['tcp_cwnd'] = {'description': 'TCP window statistics',
+                     'type': 'timeseries',
+                     'axis_labels': ['Packets'],
+                     'series': [
+                         {
+                          'data': glob("sock_stats::**"),
+                          'label': 'TCP window size',
+                          'raw_key' : glob("**::tcp_cwnd")
+                      },
+                     ]}
```

Comment:
On top, I tend to decist from mixing active code with configuration as done in tcp_stats.inc.